{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "9c2ccafa_6e5c76dd",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 6
      },
      "lineNbr": 0,
      "author": {
        "id": 27582
      },
      "writtenOn": "2022-12-05T06:09:31Z",
      "side": 1,
      "message": "recheck",
      "revId": "38c25fb5dda00fed200cf9b541c2729bbe355358",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "32fec8ec_469d4116",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 6
      },
      "lineNbr": 0,
      "author": {
        "id": 4146
      },
      "writtenOn": "2022-12-06T18:54:13Z",
      "side": 1,
      "message": "recheck",
      "revId": "38c25fb5dda00fed200cf9b541c2729bbe355358",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "7fc28bfe_abc9ef06",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 6
      },
      "lineNbr": 0,
      "author": {
        "id": 4146
      },
      "writtenOn": "2022-12-06T20:48:41Z",
      "side": 1,
      "message": "recheck not python3.11 specific since the 3.8 job failed on the latest retry. I\u0027ve got meetings so can\u0027t debug further.",
      "revId": "38c25fb5dda00fed200cf9b541c2729bbe355358",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "60aeedab_b4e6b98b",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 6
      },
      "lineNbr": 0,
      "author": {
        "id": 27582
      },
      "writtenOn": "2022-12-07T08:54:58Z",
      "side": 1,
      "message": "recheck",
      "revId": "38c25fb5dda00fed200cf9b541c2729bbe355358",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "4aecc509_271049a0",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 6
      },
      "lineNbr": 0,
      "author": {
        "id": 4146
      },
      "writtenOn": "2022-12-09T16:25:24Z",
      "side": 1,
      "message": "recheck nodepool testing should be good now",
      "revId": "38c25fb5dda00fed200cf9b541c2729bbe355358",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}